dev-qt/prison/prison-5.41.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt API for generating QRCode & DataMatrix barcodes"
+DESCRIPTION="prison is a Qt-based barcode abstraction layer/library that \
+provides uniform access to generation of barcodes with data. The API supports \
+two types of barcodes: QRCode and DataMatrix."
+HOMEPAGE="https://api.kde.org/frameworks/prison/html/index.html"
+COPYRIGHT="2010-2017 KDE Foundations"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/frameworks/${portVersion%.*}/prison-$portVersion.tar.xz"
+CHECKSUM_SHA256="b1ef221964d479255ac1ded2a96b73f90723b8e69f3b3c9a86dd1f02860a7960"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=$portVersion
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	prison$secondaryArchSuffix = $portVersion
+	lib:libkf5prison$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libdmtx$secondaryArchSuffix
+	lib:libqrencode$secondaryArchSuffix
+	lib:libqt5$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	prison${secondaryArchSuffix}_devel = $portVersion
+	devel:libkf5prison$secondaryArchSuffix = $libVersion
+	"
+REQUIRES_devel="
+	prison$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libdmtx$secondaryArchSuffix
+	devel:libqrencode$secondaryArchSuffix
+	devel:libqt5$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	extra_cmake_modules >= 5.41.0
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	constantSystemDataDir="`finddir B_SYSTEM_DATA_DIRECTORY`"
+	cmake . \
+		-DECM_DIR="$constantSystemDataDir"/cmake/Modules/ECM/cmake \
+		-DECM_MKSPECS_INSTALL_DIR=$dataDir/Qt5/mkspecs \
+		-DCMAKE_INSTALL_PREFIX:PATH=$prefix \
+		-DCMAKE_INSTALL_LIBDIR:PATH=$libDir \
+		-DCMAKE_INSTALL_INCLUDEDIR:PATH=$includeDir
+ 	make
```

Comment:
Oops, there's a space before the tab (in line 59). Could you please remove it? Thanks!

Reply:
Okay :^)

Reply:
Cool.
(Sorry for the double message ;-)